backend/app/services/implementations/residents_service.py

```diff
@@ -81,13 +82,13 @@ def delete_resident(self, resident_id):
             )
         db.session.commit()
 
-    def get_resident(self, resident_id=None):
+    def get_residents(self, resident_id=None):
```

Comment:
oopsie daisy, u forgor to change the name in the interface

Reply:
I forgor 💀